src/modules/Item.ts

```diff
@@ -0,0 +1,66 @@
+import produce from 'immer'
+import {getType} from 'typesafe-actions'
+import {
+	startFetching,
+	fetchingSuccess,
+	endWithError,
+	endCanceling,
+	startSaving,
+	savingSuccess,
+} from './commons/common'
+import useModuleEpic from './commons/moduleEpics'
+
+import ModelState from '../models/bases/ModelState'
+import {AnyAction} from 'redux'
+import Item from '../models/Item'
+
+// ------------------------------------
+// Actions
+// ------------------------------------
+
+const moduleName = 'item'
+
+export const {moduleActions, moduleEpics: itemEpics} = useModuleEpic(moduleName)
+const {getAsync, postAsync} = moduleActions
+
+export const createItem = (teamId: string, item: Item) => {
+	return postAsync.request({
+		path: `api/teams/${teamId}/expenseItems`,
```

Comment:
I haven't seen API for this so dunno if this is to create income or expense item

Reply:
It will create income or expense, depends on the category

Reply:
I think it is better to have this route as `api/teams/${teamId}/items`

Reply:
Will update it in other PR